domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/service/EntityToResponseDTOMapperTest.java

```diff
@@ -1869,6 +1878,46 @@ void testMapEmptySearchPartyEntity() {
 
     }
 
+    @Nested
+    @DisplayName("Should return Category model whose fields match those in the CategoryEntity")
+    class MapCategoryTests {
+
+
+        @Test
+        void testMapCategoryEntity() {
+            CategoryEntity categoryEntity = new CategoryEntity();
+            categoryEntity.setCaseType(caseTypeLiteEntity("CaseTypeReference"));
```

Comment:
For this test we should set all the fields of the `categoryEntity`, otherwise it's not really proving that it works